fern/versions/public.yml

```
@@ -113,6 +113,33 @@ navigation:
           - page: Snap-in triggered by an external source
             slug: triggered-external-source
             path: ../docs/pages/tutorials/triggered-external-source.mdx
+      - section: ADaaS
+        slug: adaas
+        contents:
+          - page: "Airdrop-as-a-Service (ADaaS)"
+            slug: overview
+            path: ../docs/pages/adaas/overview.mdx
+          - page: "Getting Started"
```

Sentence case on titles. https://www.notion.so/devrev/Which-words-in-a-heading-or-title-should-be-capitalized-81984d54dd414af5b343bce1327c05bd?pvs=4
```suggestion
          - page: "Getting started"
```